README.md

```diff
@@ -1,2 +1,24 @@
 # DevTeamCharter
 
+## Mission
+The Dev Team are here to solve the problems of the Scottish Tech Army by delivering usable software. In turn this will help people during the current crisis and recovery brought on by the Covid-19 outbreak.
+
+## The Team
+The team is made up of skilled technologists, for the most part software developers with the capability to contribute to applications that will be deployed on the internet. An individuals value to the team will be measured primarily by their contribution.
```

Comment:
The team is made up of a mixture of enthusiastic volunteers with a mix of different skilled technologists, for the most part software developers with the capability to contribute to applications that will be deployed on the internet. An individuals value to the team will be measured primarily by their contribution.